src/SFA.DAS.AssessorService.Api.Types/Models/Certificates/GeContacttPreviousAddressesRequest.cs

```diff
@@ -0,0 +1,10 @@
+﻿using MediatR;
+using SFA.DAS.AssessorService.Domain.Entities;
+
+namespace SFA.DAS.AssessorService.Api.Types.Models.Certificates
+{
+    public class GetContactPreviousAddressesRequest : IRequest<CertificateAddress>
```

Comment:
Can you fix the typo on this file please?

Reply:
sure np

Reply:
done